api_tests/lib/setup_chai.ts

```diff
@@ -1,21 +1,21 @@
 // A helper file to configure chai with all the plugins we need
 // This is to reduce noise in our actual test files
 
-import chaiBn = require("bn-chai");
 import { tv4, use } from "chai";
 import chaiAsPromised from "chai-as-promised";
+// @ts-ignore
```

Comment:
:mega: Is there a way how we can _not_ introduce this? I'd like to have more verification at compile-time than runtime failures.

Reply:
I don't know if there is a way to not introduce this which does not involve a 100 lines typing module.